.gitignore

```diff
@@ -6,7 +6,6 @@
 !.vscode/extensions.json
 /wled00/Release
 /wled00/extLibs
-/platformio_override.ini
```

Comment:
I think that all additional boards should be included in platformio.ini, so that ..._override.ini, that's why this file is in .gitignore